bitnami/mariadb-galera/values.yaml

```diff
@@ -239,9 +239,12 @@ galera:
   ## ref: https://github.com/bitnami/bitnami-docker-mariadb-galera#bootstraping
   ##
   bootstrap:
+    ## @param galera.bootstrap.forceBootstrap Option to force the boostraping from the indicated node in `galera.bootstarp.bootstrapFromNode`
+    ##
+    forceBootstrap: false
```

Comment:
I'm a little bit confused. Could you please explain the difference between `galera.bootstrap.forceBootstrap` and `galera.bootstrap.forceSafeToBootstrap`?

Reply:
Sure.
- `forceBootstrap` is to force a cluster boostraping from the node indicated in `bootstrapFromNode`. This action is only needed when the cluster is stopped and you need to start it again from a particular node.
- `forceSafeToBootstrap` is to set the flag `safe_to_bootstrap: 1` in the file `/bitnami/mariadb/data/grastate.dat. This is a mariadb file where some info regarding the galera node is stored. That flag indicates if it is safe to bootstrap from the node. This indicates if the node is the most up to date, so it is safe to start this node first and let the other sync from this. If the cluster is abruptly stopped the nodes aren't able to know if they are the latest or not hence all will have '0'. To start it again  you will need to force this to '1' in the node you want to bootstrap from. [Here](https://galeracluster.com/2016/11/introducing-the-safe-to-bootstrap-feature-in-galera-cluster/) you have more info on this topic.